homeassistant/components/rfxtrx/strings.json

```diff
@@ -70,5 +70,16 @@
       "invalid_input_off_delay": "Invalid input for off delay",
       "unknown": "[%key:common::config_flow::error::unknown%]"
     }
+  },
+  "device_automation": {
+    "action_type": {
+      "send_status": "Trigger status: {subtype}",
+      "send_chime": "Send command: {subtype}",
```

Comment:
Is this really right?

Reply:
Well send_chime may go away soonish. Dependent on what happens with: https://github.com/Danielhiversen/pyRFXtrx/pull/116

Reply:
Ah, so "send_chime" did really mean "send_command" and  Danielhiversen/pyRFXtrx#116 renames the function.
Does this PR need additional rework now that Danielhiversen/pyRFXtrx#116 is merged?